docs/Xamarin.Forms/Device.xml

```diff
@@ -125,7 +125,7 @@ Device.BeginInvokeOnMainThread (() => {
         <summary>Gets the flow direction on the device.</summary>
         <value>The flow direction on the device. See Remarks.</value>
         <remarks>
-          <para>The following contains a few important points about flow direction from <format type="text/html"><a href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/localization/rtl">Right-to-Left Localization</a></format>. Developers should consult that document for more information about limitations of right-to-left support, and for requirements to implement right-to-left support on various target platforms.</para>
+          <para>The following contains a few important points about flow direction from <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/right-to-left">Right-to-Left Localization</a></format>. Developers should consult that document for more information about limitations of right-to-left support, and for requirements to implement right-to-left support on various target platforms.</para>
```

Comment:
Let's remove the locale in this URL so the user will hit the correct page for their current locale if it exists (or fall back to this one if not).
```suggestion
          <para>The following contains a few important points about flow direction from <format type="text/html"><a href="https://docs.microsoft.com/xamarin/xamarin-forms/app-fundamentals/localization/right-to-left">Right-to-Left Localization</a></format>. Developers should consult that document for more information about limitations of right-to-left support, and for requirements to implement right-to-left support on various target platforms.</para>
```